tests/benchmarks/current_consumption/nfc_idle/Kconfig.sysbuild

```
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config REMOTE_BOARD
+	string "The board used for remote target"
```

afaict there is no reason for this field to have a prompt.

It's much cleaner (and simpler) to adjust the board qualifier based on the SoC.
This further has the benefit of supporting all boards based on this SoC and not having to create a dedicated sysbuild conf file for each board using the SoC.

Like this:
```suggestion
	string
	default "$(BOARD)/nrf54h20/cpurad" if SOC_NRF54H20_CPUAPP 
```